gulpfile.js

```diff
@@ -0,0 +1,9 @@
+const gulp = require('gulp');
+const replace = require('gulp-replace');
+
+gulp.task('replace', function() {
+  const serverUrl = 'http://localhost:3001/events';
+  return gulp.src(['./src/script.js', './src/index.html', './src/style.css'])
+    .pipe(replace('{{SERVER_URL}}', serverUrl))
```

Comment:
Might as well just use a mustache library (https://github.com/rogeriopvl/gulp-mustache)

```{js}
.pipe(mustache(process.env))
```

Then any use can put any environment variable in {{}} and it will work, with or without spacing, etc... as one will expect mustache templating to work!